samples/dnn/nano_tracker.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import print_function
+# NanoTrack
```

Comment:
I propose to extend existed sample here: https://github.com/opencv/opencv/blob/4.6.0/samples/python/tracker.py

Single sample would be useful to compare performance / results in a similar way.

Reply:
Agree. 
Should we keep this `nano_tracker.py`?

Reply:
Agree. 
Should we keep this `nano_tracker.py`?

Reply:
I believe one sample is enough.